j2/rules/govbr_controle_de_solicitacoes.yaml

```diff
@@ -0,0 +1,141 @@
+# DO NOT UPDATE THIS FILE MANUALLY; ANY CHANGES WILL BE OVERRIDDEN BY THE J2 DIR TEMPLATES
```

Comment:
Why did we create a new file for this?  It looks like a copy/paste job from the other bb_xxxx!   Please try to reuse these rules as much as possible